### PR TITLE
feat(curriculum): add all english block 1 animations

### PR DESCRIPTION
**Pull request by moT01**
This adds the videos to all of the block 1.1 challenges. I added videos in many places where there wasn't an audio reference in the markdown - they seemed to fit. When reviewing, we should make sure that we have the videos where we want them, and not where we don't.

It makes some slight tweaks in a couple other files, as well.

To test this:
-Download the english assets here: https://github.com/moT01/english-assets
-Serve the assets folder on port 8080 - I run `http-server` while in the english-assets folder
-Let me know if you need help with that

Edit: Alternatively, change the domain variable in `scene-assets.tsx` to `const domain = 'https://cdn.freecodecamp.org/curriculum/english/animation-assets';`

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

I didn't go through all the dialogues, but the random samples I took all worked great.

**Review by naomi-lgbt — APPROVED** [review]
Functionality looks good!

**Comment by moT01** [comment]
Thanks for the reviews 🎉

**Review by DanielRosa74 — CHANGES_REQUESTED** [review]
Noticed some things still needing to change, but it is great overall.

File: curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d2600b0ffa811dac3853f.md
[the remark this answers is not in the document]
Comment (moT01) [reply curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656d2600b0ffa811dac3853f.md:1]: I think this one is fine?

**Pull request merged**